frontend/src/app/projects/project-detail/model-overview/move-model/move-model.component.html

```diff
@@ -0,0 +1,48 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+<mat-card>
+  <h2>Move Model to Different Project</h2>
+  <div>Select the project to which you want to move the model to.</div>
+  <mat-form-field id="search" appearance="outline" class="w-full">
+    <mat-label>Search</mat-label>
+    <input
+      [(ngModel)]="search"
+      autocomplete="off"
+      matInput
+      placeholder="Project Name"
+    />
+    <mat-icon matSuffix>search</mat-icon>
+  </mat-form-field>
+  <mat-selection-list
```

Comment:
In this case, a mat list is also ok. As we have a scrollbar on y-overflow and the button is always visible, it's not comparable to the T4C Instance settings, where there was no scrollbar and the button was always below the long list of all repositories. 

What I don't like here is that the dialog resizes after each model selection due to the long button. 
There are two options to fix this: 
- Use the suggestion from @dominik003 and get rid of the mat-list, and use a button on each line instead. This would be also my preferred solution.
- Don't include the model and project name in the button text.
- Use a larger width of the dialog.

Configuration dialog would be optional, as the action is non-breaking (if the wrong model is moved, it can be moved back simply).

Reply:
I included it as I felt sometimes it might be nice to just double check where you put it because otherwise you might have to search your projects for it